### PR TITLE
Podtnl 1.0 (golang based CLI for kubernetes)

### PR DESCRIPTION
**Pull request by ghost**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/podtnl.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class Podtnl < Formula
+  desc "Expose your pod to Online from any kubernetes clusters"
+  homepage "https://podtnl.sh"
```
Comment (alebcay) [inline Formula/podtnl.rb:3]: 404 error when I try to reach it.

File: Formula/podtnl.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  url "https://github.com/narendranathreddythota/podtnl/archive/1.0.tar.gz"
```
Comment (alebcay) [inline Formula/podtnl.rb:4]: GitHub repository is not notable enough, considering it a self-submission..

File: Formula/podtnl.rb [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  sha256 "e94d28fae97ee38ee3f4b3a502df5c510280eb49868aeb2cad40a7266158c0cc"
+  head "https://github.com/narendranathreddythota/podtnl.git"
+  depends_on "go" => :build
+  def install
+    ENV["GOPATH"] = buildpath
```
Comment (alebcay) [inline Formula/podtnl.rb:9]: Formula should probably use modules-based `go` build instead of `GOPATH`.

**Comment by alebcay** [comment]
```
==> brew audit --skip-style podtnl --online --new-formula
==> FAILED
Error: 3 problems in 1 formula detected
podtnl:
  * The URL https://podtnl.sh is not reachable
  * GitHub repository not notable enough (<30 forks, <30 watchers and <75 stars)
  * GitHub repository too new (<30 days old)
```

File: Formula/podtnl.rb (hunk 3 above)
Comment (narendranathreddythota) [reply Formula/podtnl.rb:9]: Hello @alebcay Thank you I will correct it

File: Formula/podtnl.rb (hunk 1 above)
Comment (narendranathreddythota) [reply Formula/podtnl.rb:3]: I will have a look

**Comment by SMillerDev** [comment]
Hi, this seems like a nice software, but not notable enough for us to package in Homebrew for now. This is not a decision on the merits of the software, but we cannot distribute every single piece of software and have to draw a line on “notability” somewhere. Of course, we would be happy to reconsider in the future.

Please consider hosting it in a personal tap, which is very easy to do: https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap

**Pull request closed without merging**